### DISCUSSION TITLE
Localhost Testing?

### DISCUSSION THREAD

**Discussion by trymeouteh** [prosopo/captcha#1528] [Q&A] [answered]
How do learn and test out Procaptcha? I am unable to sign up and set the URL to localhost nor can I find any free site keys and secret keys for development testing purposes only.

**Comment by hckhanh** [comment]
For now I think that you should create another account and set the domain restriction to empty. That will be the testing key.

**Reply by trymeouteh** [reply, marked as answer]
Now I see a feature on the portal.prosopo.io/settings page to allow localhost
